### ISSUE TITLE
508-defect-4, Side nav component, [SCREENREADER]: "In this section" menu needs additional context

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#28698]
- [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Liz Lantz @andaleliz 

## User Story or Problem Statement

As a user navigating VA.gov with a screenreader, I need to know what a button does so I know whether or not I need to click on it to find what I'm looking for. 

## Details

When you navigate a page with the "In this section" menu (example: https://www.va.gov/health-care/family-caregiver-benefits/), the screenreader doesn't announce that the button will open a menu, only that it's a button.  We should add "menu" to the label so it's clear what the button does. 
https://user-images.githubusercontent.com/55992116/129371256-e4b7cfd2-6f24-4be1-bff5-288500d973ad.mov

## Acceptance Criteria

- [x] Screenreader includes the word "Menu" when reading the button

## Steps to Recreate

1. Shrink browser width to mobile, or use a mobile device
2. Turn on a screenreader (e.g. VoiceOver)
3. Navigate to a page that includes the "in this section" menu (most hub pages)
4. Tab to the "In this section" menu
5. Listen to the read out: "In this section, button, main" (needs to say "In this section menu")
-->

## Proposed Solution (if known)

Add `aria-label="In this section menu"` to the button element

**Comment by caw310** [comment]
@JoeTice , could you add this to your list of a11y items you're tackling? 
@1Copenut . would you add a priority for this compared to other issues? 

Thanks!

**Comment by 1Copenut** [comment]
@caw310 I agree with the priority set on this issue. It's not blocking use, but would benefit from an improved message. I've got all of the a11y issues captured in [vets-design-system-documentation issue #500](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/500). I can't cross-link this issue to that outside of a comment. Any suggestions to collect these in one place?

**Comment by 1Copenut** [comment]
Thank you for opening this issue @andaleliz ! We've added it to the Testing Tools Team a11y work log.

**Comment by holdenhinkle** [comment]
PR - https://github.com/department-of-veterans-affairs/vets-website/pull/18629

**Comment by humancompanion-usds** [comment]
The work on this was done 2 years ago. Closing.
